### PR TITLE
Feature/local account module

### LINKED ISSUE #3615: Create local account: display account before granting access to the network
Like we do in preregistration, it would be useful to present the created account at the end of the registration just before granting access to the network so the user can print or note down the credentials in case the email doesn't go through

### PR DESCRIPTION
**Pull request by julsemaan**
# Description
Display local account details even when not in preregistration

# Impacts
Captive portal registration

# Issue
fixes #3615 

# Delete branch after merge
YES

# Checklist
(REQUIRED) - [yes, no or n/a]
- [no] Document the feature
- [n/a] Add unit tests
- [yes] Add acceptance tests (TestLink)

# NEWS file entries
## Enhancements
* Display local account in the captive portal during registration if applicable (#3615)

### CODE REVIEW

File: html/captive-portal/lib/captiveportal/DynamicRouting/Module/ShowLocalAccount.pm [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+=cut
+
+=head1 AUTHOR
+
+Inverse inc. <info@inverse.ca>
+
+=head1 COPYRIGHT
+
+Copyright (C) 2005-2018 Inverse inc.
```
Comment (fdurand) [inline html/captive-portal/lib/captiveportal/DynamicRouting/Module/ShowLocalAccount.pm]: 2019

File: html/captive-portal/lib/captiveportal/PacketFence/DynamicRouting/Module/ShowLocalAccount.pm [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    }
+}
+
+=head1 AUTHOR
+
+Inverse inc. <info@inverse.ca>
+
+=head1 COPYRIGHT
+
+Copyright (C) 2005-2018 Inverse inc.
```
Comment (fdurand) [inline html/captive-portal/lib/captiveportal/PacketFence/DynamicRouting/Module/ShowLocalAccount.pm]: 2019

File: html/pfappserver/lib/pfappserver/Form/Config/PortalModule/ShowLocalAccount.pm [hunk 3]
```diff
@@ -0,0 +1,79 @@
+package pfappserver::Form::Config::PortalModule::ShowLocalAccount;
+
+=head1 NAME
+
+pfappserver::Form::Config::PortalModule:Choice
```
Comment (fdurand) [inline html/pfappserver/lib/pfappserver/Form/Config/PortalModule/ShowLocalAccount.pm]: pfappserver::Form::Config::PortalModule::ShowLocalAccount

File: html/pfappserver/lib/pfappserver/Form/Config/PortalModule/ShowLocalAccount.pm [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+=head1 DESCRIPTION
+
+Form definition to create or update a choice portal module.
```
Comment (fdurand) [inline html/pfappserver/lib/pfappserver/Form/Config/PortalModule/ShowLocalAccount.pm]: Form definition to create or update a show local account module.

File: html/pfappserver/lib/pfappserver/Form/Config/PortalModule/ShowLocalAccount.pm [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    $self->field('skipable')->default($self->for_module->meta->find_attribute_by_name('skipable')->default->());
+}
+
+=over
+
+=back
+
+=head1 COPYRIGHT
+
+Copyright (C) 2005-2018 Inverse inc.
```
Comment (fdurand) [inline html/pfappserver/lib/pfappserver/Form/Config/PortalModule/ShowLocalAccount.pm]: 2019

File: html/captive-portal/lib/captiveportal/PacketFence/DynamicRouting/Module/ShowLocalAccount.pm [hunk 6]
```diff
@@ -0,0 +1,72 @@
+package captiveportal::PacketFence::DynamicRouting::Module::ShowLocalAccount;
+
+=head1 NAME
+
+DynamicRouting::Module::ShowLocalAccount
+
+=head1 DESCRIPTION
+
+Module to show a message to the user
```
Comment (fdurand) [inline html/captive-portal/lib/captiveportal/PacketFence/DynamicRouting/Module/ShowLocalAccount.pm]: Module to show his Local Account Information to the user

**Comment by julsemaan** [comment]
Adjustments made

**Pull request merged**